app/scss/partials/_upgrade_banner.scss

```diff
@@ -33,7 +33,7 @@
 }
 
 .UpgradeBanner--small {
-	height: 50px;
+	height: 20px;
```

Comment:
Reverted on `feature/cleanup-cleanup`.

This change was made because the height overlaps the Tracker Wheel on the Detailed View and prevents tooltips from showing and click events from propagating correctly.  As this is now a known bug we should a Jira ticket.

Reply:
On second thought, I really don't like this bug.
I made a change on `feature/cleanup-cleanup` where I got rid of the background SVG completely and re-added the background gradient and white arrow using CSS only.  It lets us delete the app/images/panel/green-upgrade-banner-small.svg file completely and doesn't have any negative size implications.  See the examples below:

Original code when the "Upgrade To" text is long:
<img width="238" alt="Example old way - looks bad" src="https://user-images.githubusercontent.com/1928870/84563020-be521880-ad58-11ea-9655-ed736b162eb1.png">

New code where the "Upgrade To" text is long I:
<img width="257" alt="Example new way small - looks good" src="https://user-images.githubusercontent.com/1928870/84563014-b6927400-ad58-11ea-9789-1363f91e58fc.png">

New code where the "Upgrade To" text is long II:
<img width="587" alt="Example new way large - looks good" src="https://user-images.githubusercontent.com/1928870/84563018-babe9180-ad58-11ea-9ae1-709ef9efbebd.png">

Reply:
That looks much better & it's nice that we can get rid of the SVG. Thank you for doing that!